lib/car.rb

```diff
@@ -0,0 +1,16 @@
+require 'pry'
+
+class Car
+  attr_reader :car, :monthly_payment, :loan_length, :model, :make
+
+  def initialize(car, monthly_payment, loan_length)
+    @car = car
+    @monthly_payment =  monthly_payment
+    @loan_length = loan_length
+    @make, @model = @car.split(" ")
```

Comment:
This is a very efficient way to create the `@make` and `@model` attributes. Great work!